modules/ROOT/pages/configuration/configuration-settings.adoc

```diff
@@ -775,7 +775,7 @@ m|+++20m+++
 [frame="topbot", stripes=odd, grid="cols", cols="<1s,<4"]
 |===
 |Description
-a|Automatically enable free servers.
+a|Automatically enables all `Free` servers when the DBMS starts for the first time.
```

Comment:
@jackwaudby hi! I'd like to double check if this is correct. 
I assume that whenever you add a new server to the cluster and configure this setting in the _neo4j.conf_ file of the new server, this server will be automatically enabled.

Reply:
What about this description:

Automatically enables servers that are in the FREE state -- not only during the initial DBMS startup but also whenever a new server joins the cluster.